src/stores/events/EventsDataStore.ts

```diff
@@ -261,7 +261,7 @@ export default class EventsDataStore {
 			}
 		} catch (error) {
 			console.error(error);
-			if (error.name !== 'AbortError') {
+			if (error instanceof DOMException && error.code === error.ABORT_ERR) {
```

Comment:
it was checking if it isn't an AbortError, but your code does the opposite.
i think we can also extract this check to a function isAbortError since we use it often